packages/protocol/src/action-protocol/element-navigation.ts

```diff
@@ -86,91 +160,226 @@ export namespace NavigationTarget {
 
 /**
  * Action that is usually sent from the client to the server to request navigation targets for a specific navigation type such as
- * documentation or implementation in the given editor context.
+ * `documentation` or `implementation` in the given editor context.
+ * The corresponding namespace declares the action kind as constant and offers helper functions for type guard checks
+ * and creating new `RequestNavigationTargetsActions`.
  */
-export class RequestNavigationTargetsAction implements RequestAction<SetNavigationTargetsAction> {
-    static readonly KIND = 'requestNavigationTargets';
-    kind = RequestNavigationTargetsAction.KIND;
-    constructor(readonly targetTypeId: string, readonly editorContext: EditorContext, readonly requestId: string = generateRequestId()) {}
+export interface RequestNavigationTargetsAction extends RequestAction<SetNavigationTargetsAction> {
+    /**
+     * The unique action kind.
+     */
```

Comment:
I'd spare such comments, also if it hurts to be "incomplete". But imho they just add noise and add no information.
```suggestion
```